e2e-tests/one-pod/conf/on-demand-backup-s3.yml

```diff
@@ -0,0 +1,7 @@
+apiVersion: psmdb.percona.com/v1
```

Comment:
please use name and content identical to `e2e-tests/demand-backup/conf/backup-minio.yml`

Reply:
done